src/helpers/ComponentRegister.tsx

```diff
@@ -27,6 +12,14 @@ export interface ComponentRegisterComponents {
    * Defaults to `"a"`.
    */
   Link?: LinkComponent;
```

Comment:
Probably worth pointing out that if this needs changing, then it's likely `LinkButton` also needs specifying, and vice versa.